src/serviceClients/jobManagerClient.ts

```diff
@@ -0,0 +1,194 @@
+import { Logger } from '@map-colonies/js-logger';
```

Comment:
consider using mc-priority npm package as it has the api and all job data types included